cli/src/main/java/keywhiz/cli/Printing.java

```diff
@@ -154,6 +154,15 @@ public void printGroupWithDetails(Group group) {
     System.out.println(DOUBLE_INDENT + DateFormat.getDateTimeInstance().format(d));
   }
 
+  public void printDeletedSecretsWithDetails(List<SanitizedSecret> deletedSecrets) {
+    System.out.println("Deleted Secrets:");
+    if (deletedSecrets.isEmpty()) {
+      System.out.println("None found");
+    } else {
+      deletedSecrets.forEach(this::printSanitizedSecretWithDetails);
+    }
+  }
```

Comment:
I wonder if it would be helpful to include the number of deleted secrets somewhere in the output (maybe at the end of line 158?).